src/histolab/tiler.py

```diff
@@ -296,17 +300,59 @@ def tile_size(self, tile_size_: Tuple[int, int]):
 
     # ------- implementation helpers -------
 
+    def _are_coordinates_within_extraction_mask(
+        self,
+        tile_thumb_coords: CoordinatePair,
+        binary_mask_region: np.ndarray,
+    ) -> bool:
+        """Chack whether the ``tile_thumb_coords`` are inside of ``binary_mask_region``.
+
+        Return True if the tile defined by ``tile_thumb_coords`` is inside of the True
+        areas of ``binary_mask_region`` for at least 80% of its area.
```

Comment:
here I would say `Return True if a fixed portion of the tile defined by ``tile_thumb_coords`` is inside the  area of the ``binary_mask_region``